_dependencies/defaults/main.yml

```diff
@@ -1,6 +1,4 @@
 ---
-# Ansible interpreter
-ansible_python_interpreter: "/usr/bin/python3"

```

Comment:
When running a redeploy, the absence of a defined interpreter resulted in an error: 
```
[REDACTED] \"module_stderr\": \"/bin/sh: 1: /usr/bin/python: not found\\n\", \"module_stdout\": \"\", \"msg\": \"The module failed to execute correctly, you probably need to set the interpreter.\\nSee stdout/stderr for the exact error\", \"rc\": 127}",
```

Any specific reason why this line was removed?

Reply:
It should not be necessary to define the interpreter - Ansible should look it up automatically (https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html).  It was only there to _force_ Python3 for some earlier testing, and should not be there in general.  I've found that this error is a symptom of something else; did you manage to _deploy_ without the interpreter set?